Test/Test.cs

```diff
@@ -550,5 +554,126 @@ public void TestCreateOrderEventWithWirePaymentMethod()
             Assert.Equal("https://api.sift.com/v205/events?abuse_types=legacy,payment_abuse&return_score=true",
                           Uri.UnescapeDataString(eventRequest.Request.RequestUri.ToString()));
         }
+
+        [Fact]
+        public void TestVerificationCheckRequest()
```

Comment:
Have these new request types been verified by running them against the API?

Reply:
@mpierotti-sift All verification request were tested by adding the library with existing web application and verified the same